content/docs/user-guide/experiment-management/cleaning.md

```diff
@@ -0,0 +1,262 @@
+# Cleaning Up Experiments
+
+Although DVC uses minimal resources to keep track of the experiments, they may
+clutter tables and the workspace. DVC allows to remove particular experiments
+from the workspace or delete all not-yet-persisted experiments at once.
+
+## Removing Local Experiments
```

Comment:
I think "local" is assumed normally, how about "specific" since that's what `exp remove` does?

> Also, we don't use full title case except in the H1 title.

```suggestion
## Removing specific experiments
```

Reply:
BTW "experiments" is plural but the section only shows removing a single exp. Maybe add another few lines sending several targets to `exp remove` and/or state "one or more" somewhere.

Reply:
I used "local" vs. remote. "Specific" may also be remote but that use case is not covered by DVC. When I put "specific", I think I'll also have another title with "Removing All Experiments" somewhere.

Reply:
Done per your suggestion. Thanks.

Reply:
Thank you @iesahin. I can only focus on one big review at a time so I'll stick to https://github.com/iterative/dvc.org/pull/2690 for now. I moved this one from https://github.com/orgs/iterative/projects/166 to https://github.com/orgs/iterative/projects/167 for now.

If you prefer you can close it temporarily or mark it as a draft. Up to you